compiler/Eta/Main/GHC.hs

```diff
@@ -178,7 +178,7 @@ module Eta.Main.GHC (
         isRecordSelector,
         isPrimOpId, isFCallId, isClassOpId_maybe,
         isDataConWorkId, idDataCon,
-        isBottomingId, isDictonaryId,
+        isBottomingId, isDictionaryId,
```

Comment:
Again, I'm not sure if this is a public or private item. As each change is a distinct commit, if I'm wrong, dropping it isn't a big deal.